types/mdast/index.d.ts

```diff
@@ -1,346 +1,1159 @@
+/**
+ * Markdown code (flow) (block).
+ */
+export interface Code extends Literal {
+    /**
+     * Node type of mdast code (flow).
+     */
+    type: 'code';
+    /**
+     * Language of computer code being marked up.
+     */
+    lang?: string | null | undefined;
+    /**
+     * Custom information relating to the node.
+     *
+     * If the lang field is present, a meta field can be present.
+     */
+    meta?: string | null | undefined;
+    /**
+     * Data associated with the mdast code (flow).
+     */
+    data?: CodeData | undefined;
 }
 
-export interface Literal extends UnistLiteral {
-    value: string;
+/**
+ * Info associated with mdast code (flow) (block) nodes by the ecosystem.
+ */
+// tslint:disable-next-line
+export interface CodeData extends Data {}
+
+/**
+ * Markdown definition.
+ */
+export interface Definition extends Node, Association, Resource {
+    /**
+     * Node type of mdast definition.
+     */
+    type: 'definition';
+    /**
+     * Data associated with the mdast definition.
+     */
+    data?: DefinitionData | undefined;
 }
 
-export interface Root extends Parent {
-    type: 'root';
+/**
+ * Info associated with mdast definition nodes by the ecosystem.
+ */
+// tslint:disable-next-line
+export interface DefinitionData extends Data {}
+
+/**
+ * Markdown GFM delete (strikethrough).
+ */
+export interface Delete extends Parent {
+    /**
+     * Node type of mdast GFM delete.
+     */
+    type: 'delete';
+    /**
+     * Children of GFM delete.
+     */
+    children: PhrasingContent[];
+    /**
+     * Data associated with the mdast GFM delete.
+     */
+    data?: DeleteData | undefined;
 }
 
-export interface Paragraph extends Parent {
-    type: 'paragraph';
+/**
+ * Info associated with mdast GFM delete nodes by the ecosystem.
+ */
+// tslint:disable-next-line
+export interface DeleteData extends Data {}
+
+/**
+ * Markdown emphasis.
+ */
+export interface Emphasis extends Parent {
+    /**
+     * Node type of mdast emphasis.
+     */
+    type: 'emphasis';
+    /**
+     * Children of emphasis.
+     */
     children: PhrasingContent[];
+    /**
+     * Data associated with the mdast emphasis.
+     */
+    data?: EmphasisData | undefined;
+}
+
+/**
+ * Info associated with mdast emphasis nodes by the ecosystem.
+ */
+// tslint:disable-next-line
+export interface EmphasisData extends Data {}
+
+/**
+ * Markdown GFM footnote definition.
+ */
+export interface FootnoteDefinition extends Parent, Association {
+    /**
+     * Node type of mdast GFM footnote definition.
+     */
+    type: 'footnoteDefinition';
+    /**
+     * Children of GFM footnote definition.
+     */
+    children: Array<BlockContent | DefinitionContent>;
+    /**
+     * Data associated with the mdast GFM footnote definition.
+     */
+    data?: FootnoteDefinitionData | undefined;
 }
 
+/**
+ * Info associated with mdast GFM footnote definition nodes by the ecosystem.
+ */
+// tslint:disable-next-line
```

Comment:
This line occurs multiple times. Maybe it’s better to just disable this tslint rule entirely.

Reply:
done!